shared/downloads/fedora-30-aarch64.ini

```diff
@@ -0,0 +1,4 @@
+[fedora-30-aarch64]
```

Comment:
nice touch, although not sure how important (I never used Avocado to download it...)